src/third_party/progress/progress.hpp

```diff
@@ -208,6 +216,11 @@ class C_Progress_display : public C_Progress
     } // display_tic
 };
 
+inline C_Progress& C_Progress::dummy()
+{
+  static C_Progress null;
```

Comment:
Ok, thx for the info.
I think it will be simpler to keep the static reference as you done it.
I was just pointing you the call_once in order to let us know that there is new interesting C++11 feature to deal with singleton allocation in multithreaded context.